api/types/provisioning.go

```diff
@@ -461,3 +478,28 @@ func (a *ProvisionTokenSpecV2CircleCI) checkAndSetDefaults() error {
 	}
 	return nil
 }
+
+func (a *ProvisionTokenSpecV2Kubernetes) checkAndSetDefaults() error {
```

Comment:
It's worth adding a few cases to `TestProvisionTokenV2_CheckAndSetDefaults` that cover the `kubernetes` fields :)

Reply:
done